test-harness/specs/security-scheme-validation-AC-6_failing.oas3.txt

```diff
@@ -0,0 +1,34 @@
+====test====
+Adding a header of "Authorization: Basic <<TOKEN>>" where TOKEN is NOT base64 encoded <<ID>>:<<PASS>> combination
+for a request to an endpoint, which security depends on global security definitions, results in denied access.
+====spec====
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      security:
+        - BasicAuth: []
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              example: test
+components:
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+    ApiKeyAuth:
```

Comment:
Would be nice to test the `ApiKeyAuth` validation as well.

Reply:
@chris-miaskowski, I've added presence validation in https://github.com/stoplightio/prism/pull/484/commits/4a0fd4990a575d7a42d26ec28cc5e62dba5c0719#diff-0b83a74c0a2f17482bdada0815512f6f
don't know if we can other type of validation here since api key is free-form (?)